src/ark/phenotyping/pixel_meta_clustering.py

```diff
@@ -371,12 +371,22 @@ def apply_pixel_meta_cluster_remapping(fovs, channels, base_dir,
         ].values
     )
 
+    # verify each pixel meta cluster assigned to a unique renamed meta cluster
+    # duplicates cause issues with aggregation
+    pixel_meta_pairs = pixel_remapped_data[
+        ['pixel_meta_cluster', 'pixel_meta_cluster_rename']
+    ].drop_duplicates()
+    pixel_meta_dups = pixel_meta_pairs[
+        pixel_meta_pairs.duplicated('pixel_meta_cluster_rename', keep=False)
+    ]['pixel_meta_cluster_rename'].unique().tolist()
+    if len(pixel_meta_dups) > 0:
+        raise ValueError(
+            "Duplicate renamed pixel meta cluster values found: %s, "
+            "please re-run remapping GUI to resolve naming conflicts" % str(pixel_meta_dups)
+        )
```

Comment:
Since you've used this block of code for both `cell` and `pixel` meta clustering, do you think it would be valuable to refactor it into a utility function?

Reply:
On second thought, might want to think a bit more about this. We have separate `pixel_cluster_utils` and `cell_cluster_utils`, but no module that includes functionality shared by both pixel and cell.

I'm a bit hesitant to use `cluster_helpers`, since that was intended for just the `Pixie` class definitions. However, since SOM and meta clustering processes are shared for both pixel and cell, we could potentially just plop shared functions there anyways.

Open to discussion.